packages/ui/src/providers/Locale/index.tsx

```diff
@@ -12,73 +12,86 @@ import { usePreferences } from '../Preferences/index.js'
 
 const LocaleContext = createContext({} as Locale)
 
+export const LocaleLoadingContext = createContext({
+  localeIsLoading: false,
+  setLocaleIsLoading: (_: boolean) => {},
```

Comment:
Is this type correct? I think it returns void

Reply:
That is an initialized function, not a type, but yea it should return `undefined` and not an empty object. Fixed.